WBooks/Library/Details/BookInfo/BookInfoViewController.swift

```diff
@@ -0,0 +1,92 @@
+//
+//  BookInfoViewController.swift
+//  WBooks
+//
+//  Created by ana.mancuso on 17/10/2022.
+//
+
+import UIKit
+
+class BookInfoViewController: UIViewController {
+    
+    //MARK: Properties
+    
+    private lazy var bookView = BookInfoView()
+    private let bookModel: BookInfoViewModel
+    
+    //MARK: Initialization
+    
+    init (bookModel: BookInfoViewModel) {
+        self.bookModel = bookModel
+        super.init(nibName: nil, bundle: nil)
+    }
+    
+    required init? (coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configBook()
+    }
+    
+    override func loadView() {
+        view = bookView
+    }
+    
+    func configBook() {
+        setupData(library: bookModel.bookDetails)
+        bookView.rentButton.addTarget(self, action:#selector(rentButton), for: .touchUpInside)
+        bookView.rentButton.applyGradient(colors: ((bookView.availabilityBook.text == "Available") ? [UIColor.lightSeaGreen.cgColor, UIColor.summerSky.cgColor,  UIColor.mediumTurquoise.cgColor] : [UIColor.gainsboro.cgColor]))
```

Comment:
the logic of knowing if it is available or not should be in a function in the ViewModel for example
`        bookView.rentButton.applyGradient(colors: (bookModel.isAvailable() ? [UIColor.lightSeaGreen.cgColor, UIColor.summerSky.cgColor,  UIColor.mediumTurquoise.cgColor] : [UIColor.gainsboro.cgColor]))
`
that function should return a boolean depending on whether or not it is available

Reply:
I tried to solve the previous comment and this comment, but now it doesn't show me the button itself.